arangod/Pregel/Utils.h

```diff
@@ -141,7 +143,8 @@ class Utils {
   static int64_t countDocuments(TRI_vocbase_t* vocbase,
                                 std::string const& collection);
 
-  static ErrorCode resolveShard(ClusterInfo& ci, WorkerConfig const* config,
+  static ErrorCode resolveShard(ClusterInfo& ci,
+                                std::shared_ptr<WorkerConfig const> config,
```

Comment:
Just a heads up: I had to move this function into a new file to get rid of all the arango dependencies this Utils.h file: https://github.com/arangodb/arangodb/pull/18262/files commit f1d80710a4509a46d646b7d6034d0dac09b58866, but this PR is most probably merged earlier, so I will need merge that code together :)